Source/Interface/Tips.js

```diff
@@ -170,15 +172,22 @@ this.Tips = new Class({
 		clearTimeout(this.timer);
 		this.timer = (function(){
 			this.container.empty();
-
+			var showTip = !this.options.hideEmpty;
```

Comment:
Not a blocker for shipping this, but you declare this variable just to reference it once in a conditional; I'd remove this line and reference the option directly on line 186.